Database/Corrections/sodQuestFixes.lua

```diff
@@ -2925,6 +2925,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -402852,
             [questKeys.zoneOrSort] = sortKeys.PRIEST,
         },
+        [90188] = {
+            [questKeys.name] = "Shadow Word: Death",
+            [questKeys.startedBy] = {{209742}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
```

Comment:
[Dispel Magic](https://www.wowhead.com/classic/spell=527/dispel-magic) can not be learned before level 18.

```suggestion
            [questKeys.requiredLevel] = 18,
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5564/commits/18ba4a28a327a91e6b4f7407d4089abb46b254db